apps/rush-lib/src/cli/actions/ListAction.ts

```diff
@@ -1,11 +1,15 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+// eslint-disable-next-line
```

Comment:
Can you disable the specific rule for these?

Reply:
Sure.